deploy/onos-operator.yaml

```diff
@@ -79,6 +79,16 @@ spec:
                 x-kubernetes-preserve-unknown-fields: true
           status:
             type: object
+            properties:
```

Comment:
Actually, I just realized we're missing the `status` subresource on this CRD, e.g.:

https://github.com/atomix/runtime/blob/master/controller/helm/crds/atomix.io/storageprofiles.yaml#L26

I wonder if this is why the default value did not work. Actually, I wonder if it's validating the status at all.